### PR TITLE
[WebApp] maintenance packages upgrade

### PR DESCRIPTION
**Pull request by Maks19**
**Description:** Web-App maintenance packages upgrade

**Issue:** https://www.notion.so/saladtech/WebApp-Maintenance-7880e338587a4bfe84bd25246900415c

### CODE REVIEW

File: packages/web-app/.vscode/settings.json [hunk 1]
```diff
@@ -41,5 +51,10 @@
     "editor.codeActionsOnSave": {
       "source.organizeImports": true
     }
+  },
+  "[typescript][typescriptreact]": {
+    "editor.codeActionsOnSave": {
+      "source.organizeImports": "explicit"
+    }
```
Comment (rners01) [inline packages/web-app/.vscode/settings.json:58]: what this addition for?

File: packages/web-app/.yarn/sdks/typescript/lib/tsserver.js [hunk 2]
```diff
@@ -61,14 +61,30 @@ const moduleWrapper = tsserver => {
           //
```
Comment (rners01) [inline packages/web-app/.yarn/sdks/typescript/lib/tsserver.js:61]: is this the auto change?

File: packages/web-app/.yarn/sdks/typescript/lib/tsserver.js [hunk 3]
```diff
@@ -61,14 +61,30 @@ const moduleWrapper = tsserver => {
...
           // Ref: https://github.com/microsoft/vscode/issues/105014#issuecomment-686760910
           //
```
Comment (rners01) [inline packages/web-app/.yarn/sdks/typescript/lib/tsserver.js:63]: is this the auto change?

File: packages/web-app/.vscode/settings.json (hunk 1 above)
Comment (Maks19) [reply packages/web-app/.vscode/settings.json:58]: It's auto changed

File: packages/web-app/.yarn/sdks/typescript/lib/tsserver.js (hunk 2 above)
Comment (Maks19) [reply packages/web-app/.yarn/sdks/typescript/lib/tsserver.js:61]: It's auto changed

File: packages/web-app/.yarn/sdks/typescript/lib/tsserver.js (hunk 3 above)
Comment (Maks19) [reply packages/web-app/.yarn/sdks/typescript/lib/tsserver.js:63]: It's auto changed

**Review by rners01 — APPROVED** [review]

**Pull request merged**